frontend/src/StartView.svelte

```diff
@@ -98,6 +98,12 @@
   .clickable {
     cursor: pointer;
   }
+  .experiment-features-check {
+    position: fixed;
+    top: 3em;
+    left: 3em;
+  }
```

Comment:
I don't really understand why position in the top left like this? Feels kind of misaligned/out of place with the rest of the interface.

Reply:
I felt it shouldn't be literally front & center, so I wanted it out of the way in the corner. The bottom corners were tricky because the animal popups would overlap it and I was trying to get this done quick so I didn't bother with working through that, and just chose a top corner. 

If you think there's a better spot on the page we can quickly move this to, then let's do it. But if it's not perfect, I plan to iterate on this anyway with a button to access the full settings panel from the StartView, so this checkbox is "temporary" (though not with a definite lifetime).

Reply:
Given that this is temporary, I think it is okay. I would otherwise work to find a better spot if it was going to be in the UI long-term. 

Probably worth adding a comment noting that this is a short-term thing, though. Don't want it to slip through the cracks on the project management side and not have that noted somewhere in the code.